Documentation/building/linux-instructions.md

```diff
@@ -1,14 +1,61 @@
 Build CoreCLR on Linux
 ======================
 
-This guide will walk you through building CoreCLR on Linux.  We'll start by showing how to set up your environment from scratch.
+This guide will walk you through building CoreCLR on Linux.  Before building there is environment setup that needs to happen to pull in all the dependencies required by the build.  There are two suggested ways to go about doing this. First you are able to use the docker environments provided by https://github.com/dotnet/dotnet-buildtools-prereqs-docker, or you can set up the environment yourself. The documentation will go over both ways of building; however, in general prefer docker if you are able. 
+
+[Build using docker](#Build-using-docker)
+
+[Build with own environment](#Environment)
+
+Build using docker
+==================
+
+Install Docker, see https://docs.docker.com/install/linux/docker-ce/ubuntu/ 
```

Comment:
This of course is only for Ubuntu. For Mac (documented elsewhere) or non-Ubuntu Linux, there is another location.